pkg/dockerregistry/server/prune/prune.go

```diff
@@ -229,7 +229,7 @@ func Prune(ctx context.Context, registry distribution.Namespace, registryClient
 			return fmt.Errorf("failed to parse the repo name %s: %v", repoName, err)
 		}
 
-		ref, err := imageapi.ParseDockerImageReference(repoName)
+		ref, err := imageref.Parse(repoName)
```

Comment:
Should it be imageutil.ParseDockerImageReference?

Reply:
Indeed imageref.Parse() is a better choice. I'll replace them.